File: Practice/src/introduction_coding_tests/RemoveDuplicateCharacters.js
function solution(my_string) {
    var answer = '';
    for (let s of my_string.split('')) {
        if (!answer.includes(s)) answer += s;
    }
    return answer;
    // return [...new Set(my_string)].join('');
    // return [...my_string].filter((c, i) => s.indexOf(c) == i).join('');
}

File: Practice/src/introduction_coding_tests/RemoveDuplicateCharacters.java
package introduction_coding_tests;

public class RemoveDuplicateCharacters {
//	중복된 문자 제거
	public String solution(String my_string) {
		String answer = "";
		for (String i : my_string.split("")) {
			if (!answer.contains(i))
				answer += i;
		}
		return answer;
//		return Arrays.stream(my_string.split("")).distinct().collect(Collectors.joining());
//		return my_string.chars().mapToObj(Character::toString).distinct().collect(Collectors.joining());
	}
}